### ISSUE TITLE
TileEntity Error - Client game crash while looking on the handheldscreen

### ISSUE THREAD

**Issue by PumaBlack** [XCompWiz/LookingGlass#21]
here is the log, i hope, you can give a hint or try to fix it. Thanks in advance!

http://pastebin.com/jWzvjXE0

**Comment by Guichaguri** [comment]
This crash only points to TF2Teleporter, there's nothing to do with LookingGlass

**Comment by PumaBlack** [comment]
Only WITH Looking Glas activ, the Error occurs. There is a kind of incompatibly between both. Is there a Way to fix it?

**Comment by Guichaguri** [comment]
Yes, but I think it has to be changed in TF2Teleporter, not in LookingGlass
I was thinking TF2Teleporter register their tile entities outside the tick/update method (readFromNBT, maybe), the developer needs to register the teleporters using the update method to make it work

**Comment by XCompWiz** [comment]
Was another issue created for this elsewhere? Was it resolved?
